applications/nucleo_io/targets/freertos.armv6m.st-stm32f091rc-nucleo-dev-board/main.cxx

```diff
@@ -337,6 +337,21 @@ constexpr const Gpio *const kPortDEGpio[] = {
 openlcb::MultiConfiguredConsumer portde_consumers(stack.node(), kPortDEGpio,
     ARRAYSIZE(kPortDEGpio), cfg.seg().portde_consumers());
 
+// Rick Lull - April 15 2020
+// Snap switches and LED lights conflict on same port. When GPIO pin has
+// snap configuration in place, LED will quickly flash on consumer event recv and
+// not stay on as desired/needed for signal driver.
+// When PORTD_SNAP is set to 1 (enabled), we will set portD to be used for snap
```

Comment:
Please move this documentation to the config.hxx where the ifdef is for PORTD_SNAP